spring-amqp/src/main/java/org/springframework/amqp/core/MessageProperties.java

```diff
@@ -55,11 +55,11 @@
 
 	public static final String X_DELAY = "x-delay";
 
-	static final String DEFAULT_CONTENT_TYPE = CONTENT_TYPE_BYTES;
+	protected static final String DEFAULT_CONTENT_TYPE = CONTENT_TYPE_BYTES;
 
-	static final MessageDeliveryMode DEFAULT_DELIVERY_MODE = MessageDeliveryMode.PERSISTENT;
+	protected static final MessageDeliveryMode DEFAULT_DELIVERY_MODE = MessageDeliveryMode.PERSISTENT;
 
-	static final Integer DEFAULT_PRIORITY = Integer.valueOf(0);
+	protected static final Integer DEFAULT_PRIORITY = Integer.valueOf(0);
```

Comment:
I'd make these constant as `public`.
Even if they aren't used outside the class, they may be useful for target applications in some cases.

That's why I said let's apply the fix, but wisely...

(If you don't have any objections no reason to comment. I let you know what to do in the end of review)